govcd/lbservicemonitor.go

```diff
@@ -0,0 +1,196 @@
+/*
+ * Copyright 2019 VMware, Inc.  All rights reserved.  Licensed under the Apache v2 License.
+ */
+
+package govcd
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+)
+
+// CreateLBServiceMonitor creates a load balancer service monitor based on mandatory fields. It is a synchronous
+// operation. It returns created object with all fields (including ID) populated or an error.
+func (eGW *EdgeGateway) CreateLBServiceMonitor(lbMonitorConfig *types.LBMonitor) (*types.LBMonitor, error) {
+	if err := validateCreateLBServiceMonitor(lbMonitorConfig); err != nil {
+		return nil, err
+	}
+
```

Comment:
Please check that the edge gateway has advanced networking enabled and return an appropriate error message if it doesn't

Reply:
Added.